paramspider.py

```diff
@@ -114,9 +114,6 @@ def main():
         else:
             black_list.append("." + args.exclude)
 
-    else:
-        black_list = []  # for blacklists
-    if args.exclude:
```

Comment:
If you remove else statement above, the code looks something like this:
```py
...
if args.exclude:
    if "," in args.exclude:
        black_list = args.exclude.split(",")
        for i in range(len(black_list)):
            black_list[i] = "." + black_list[i]
    else:
        black_list.append("." + args.exclude)
if args.exclude:
    print(
        f"\u001b[31m[!] URLS containing these extensions will be excluded from the results   : {black_list}\u001b[0m\n"
    )
...
```
So I find it better to remove this secondary if statement